slices/libpython3.8-stdlib.yaml

```
@@ -0,0 +1,381 @@
+      /usr/lib/python3.8/telnetlib.py:
+      /usr/lib/python3.8/uuid.py:
+      /usr/lib/python3.8/webbrowser.py:
+      /usr/lib/python3.8/wsgiref/**:
+      /usr/lib/python3.8/xmlrpc/**:
+
+  # Networking and Interprocess Communication
+  # https://docs.python.org/3.8/library/ipc.html
+  ipc:
+    essential:
+      - libpython3.8-stdlib_core
+      - libpython3.8-stdlib_concurrency
+    contents:
+      /usr/lib/python3.8/asynchat.py:
+      /usr/lib/python3.8/asyncio/**:
+      /usr/lib/python3.8/asyncore.py:
+      /usr/lib/python3.8/lib-dynload/_asyncio.cpython-38-*-linux-*.so:
+
+  # Python Language Services
+  # https://docs.python.org/3.8/library/language.html
+  language:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/lib-dynload/parser.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/pickletools.py:
+      /usr/lib/python3.8/pyclbr.py:
+      /usr/lib/python3.8/symbol.py:
+      /usr/lib/python3.8/symtable.py:
+      /usr/lib/python3.8/tabnanny.py:
+
+  # Structured Markup Processing Tools (HTML, XML)
+  # https://docs.python.org/3.8/library/markup.html
+  markup:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/_markupbase.py:
+      /usr/lib/python3.8/html/**:
+      /usr/lib/python3.8/xml/**:
+
+  # Multimedia Services
+  # https://docs.python.org/3.8/library/mm.html
+  multimedia:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/aifc.py:
+      /usr/lib/python3.8/chunk.py:
+      /usr/lib/python3.8/colorsys.py:
+      /usr/lib/python3.8/imghdr.py:
+      /usr/lib/python3.8/lib-dynload/audioop.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/lib-dynload/ossaudiodev.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/sndhdr.py:
+      /usr/lib/python3.8/sunau.py:
+      /usr/lib/python3.8/wave.py:
+
+  # Internet Data Handling
+  # https://docs.python.org/3.8/library/netdata.html
+  net-data:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/binhex.py:
+      /usr/lib/python3.8/json/**:
+      /usr/lib/python3.8/lib-dynload/_json.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/mailbox.py:
+      /usr/lib/python3.8/mailcap.py:
+
+  # Numeric and Mathematical Modules
+  # https://docs.python.org/3.8/library/numeric.html
+  numeric:
+    essential:
+      - libpython3.8-stdlib_core
+      - libmpdec2_libs
+    contents:
+      /usr/lib/python3.8/_pydecimal.py:
+      /usr/lib/python3.8/decimal.py:
+      /usr/lib/python3.8/fractions.py:
+      /usr/lib/python3.8/lib-dynload/_decimal.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/numbers.py:
+      /usr/lib/python3.8/statistics.py:
+
+  # Shared OS X support functions
+  osx-support:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/_osx_support.py:
+
+  # Data Persistence
+  # https://docs.python.org/3.8/library/persistence.html
+  persistence:
```

```suggestion
  data-persistence:
```

(being explicit helps here)